components/table/hooks/usePagination.ts

```diff
@@ -101,8 +101,6 @@ export default function usePagination(
       pageSize,
     });
 
-    onChange(1, pageSize);
```

Comment:
It seems onChange should not be remove ?
setInnerPagination has set current to 1.